content/03.text-mining.md

```diff
@@ -0,0 +1,49 @@
+## Relationship Extraction
+
+1. Set up the context for relationship extraction
+	1. Define the problem
+	2. Talk about the importance of the problem (filling knowledge bases -> point researchers to relevant papers)
+2. Give overview towards taxonomy of approaches (hand written rules, unsupervised machine learning, supervised machine learning etc.)
```

Comment:
I think you will want another subsection on things like database construction & manual curation (non-text mining).

Are there approaches to building biomedical knowledge graphs beyond text mining or the database-based strategy?